### ISSUE TITLE
Direct Transport is sending to multiple recipients one by one and not in parallel

### ISSUE THREAD

**Issue by talkol** [orbs-network/orbs-network-go#271]
Both in broadcast and multicast, we send one by one instead of sending to all of them together

This is a performance issue

**Comment by talkol** [comment]
Reported by @netoneko

**Comment by talkol** [comment]
Issue merged into https://github.com/orbs-network/orbs-network-go/issues/273
